### PR TITLE
pulled and removed default elements and installed npm

### PR DESCRIPTION
**Pull request by NimaHBB**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,18 +5,7 @@ function App() {
   return (
```
Comment (ThomasF85) [inline src/App.js:5]: You can also remove unused imports.

**Review by ThomasF85 — CHANGES_REQUESTED** [review]
I am missing a prettierrc file

File: src/App.js (hunk 1 above)
Comment (NimaHBB) [reply src/App.js:5]: i thought maybe replace the logo later with my logo! :P
i remove it now.

Comment (NimaHBB) [reply src/App.js:4]: prettier is installed and unused import is removed.

File: package.json [hunk 2]
```diff
@@ -34,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "prettier": "2.6.2"
```
Comment (ThomasF85) [inline package.json:39]: You only need to install prettier if you define scripts like "prettier:check" in your package.json. Prettier works without this dependencies in your vs code extension.

**Review by ThomasF85 — APPROVED** [review]

**Pull request closed without merging**